README.md

```diff
@@ -28,6 +28,14 @@ versions of the supported compilers) likely won't work.
 CRADLE is built using CMake and requires a relatively recent version. It's
 known to work on 3.18 and 3.19. It should work on more recent versions as well.
 
+### vcpkg
+
+CRADLE uses vcpkg to manage dependencies. If you don't already have vcpkg,
+visit https://vcpkg.io/en/getting-started and following the instructions in the
+'Install vcpkg' section for your OS. You don't need to follow any of the other
+instructions, but you should note the path at which you installed vcpkg as
```

Comment:
A vcpkg directory tree seems to be tied to a single repository (so cannot be used across repositories). The most natural vcpkg installation path therefore would be the root of the CRADLE repository, which is also what https://vcpkg.io/en/getting-started seems to recommend ("clone vcpkg as a submodule to an existing project").

An implication of this decision is that vcpkg should be in .gitignore.